apps/src/maze/dirtDrawer.js

```diff
@@ -62,7 +62,7 @@ export default class DirtDrawer extends Drawer {
    * @param {number} val
    * @return {number}
    */
-  spriteIndexForDirt_(val) {
+  static spriteIndexForDirt_(val) {
```

Comment:
Nice... I think this is the first time I've seen someone use the `static` keyword in JavaScript.